### ISSUE TITLE
Pre-trained SpyNet

### ISSUE THREAD

**Issue by FrauSong** [JingyunLiang/VRT#5]
Hi,

Thanks for sharing the impressive paper and the code. I am curious about the SpyNet during training. Did you use pre-trained SpyNet then enable the gradient to fine-tune the weights to fit your tasks? I am not sure if it is possible to train a SpyNet from scratch together with other modules in the network. 
 
Many thanks!

Best regards
